### PR TITLE
Add approved copy for Bol.com case study

### PR DESCRIPTION
**Pull request by pymander**
This adds a new case study for Bol.com and adds a few more terms to
the spelling file.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[x ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -109,12 +110,13 @@ AssemblyScript
 Atlassian
 AttributeGen
 Auth0
+AuthPolicy
 AuthenticationPolicy
 AuthorizationPolicy
+Autoscalers
```
Comment (ericvn) [inline .spelling:116]: This is now out of order (negate all Caps happen before lowercase).

File: .spelling [hunk 2]
```diff
@@ -109,12 +110,13 @@ AssemblyScript
 Atlassian
 AttributeGen
 Auth0
+AuthPolicy
```
Comment (ericvn) [inline .spelling:113]: Also out of order.

File: content/en/about/case-studies/bol.com/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    name: "Roland Kool"
+companyName: "bol.com"
+companyURL: "https://bol.com/"
+logo: "/logos/bol-com.svg"
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
```
Comment (craigbox) [inline content/en/about/case-studies/bol.com/index.md:15]:
```suggestion
type: case-studies
weight: 30
```

File: content/en/about/case-studies/bol.com/index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Istio's [multi-cluster deployment model](/docs/ops/deployment/deployment-models/#multiple-clusters) ended up being the perfect solution. [Authorization policies](/docs/reference/config/security/authorization-policy/) could be used to securely allow workloads from different clusters to talk to each other. With Istio, Kool's team was able to move away from OSI layer 3 or 4 network policies to [authz policies](/docs/tasks/security/authorization/authz-http/) implemented in layer 7. This move was made possible by Istio's strong identity support, service-to-service authentication, and security with mutual TLS (mTLS).
+
+These changes gave bol.com the ability to scale by adding new Kubernetes clusters while maintaining service discovery, load balancing, and required security policies.
+
+## Why Istio?
+
+When bol.com initially began migrating to Kubernetes, Istio was just in version 0.2. It did not seem to be ready for production, so they went forward without Istio. They first started to look seriously at Istio around version 1.0, but they ran into too many issues with deployment and implementation. Without an urgent use case, they tabled the idea.
```
Comment (craigbox) [inline content/en/about/case-studies/bol.com/index.md:42]: Reads weird to me.  was "only at" rather than was "just in"?

File: .spelling (hunk 2 above)
Comment (pymander) [reply .spelling:113]: Is there a possibility that `make lint` could check this?

File: content/en/about/case-studies/bol.com/index.md (hunk 4 above)
Comment (pymander) [reply content/en/about/case-studies/bol.com/index.md:43]: Do you want this edit made? Is this the last one?

Comment (craigbox) [reply content/en/about/case-studies/bol.com/index.md:43]: Yes, assuming you're happy with my suggestion.

**Review by craigbox — APPROVED** [review]

**Pull request merged**